### PR TITLE
EDSC-2337: Adjust display and order payloads for excluded granules

### PR DESCRIPTION
**Pull request by abbottry**

### CODE REVIEW

File: static/src/js/actions/granules.js [hunk 1]
```diff
@@ -147,6 +147,7 @@ export const fetchLinks = retrievalCollectionData => (dispatch, getState) => {
  * @param {Object} retrievalCollectionData Retreival Collection response from the database
  */
 export const fetchOpendapLinks = retrievalCollectionData => (dispatch, getState) => {
+  console.log('asdf')
```
Comment (abbottry) [inline static/src/js/actions/granules.js]: Remove

File: static/src/js/actions/project.js [hunk 2]
```diff
@@ -124,7 +125,11 @@ export const getProjectCollections = () => (dispatch, getState) => {
 
   const { authToken } = getState()
 
+  const collectionParams = prepareCollectionParams(getState())
+  const searchParams = buildCollectionSearchParams(collectionParams)
+
   const response = getCollectionMetadata({
+    ...searchParams,
```
Comment (abbottry) [inline static/src/js/actions/project.js]: Remove filtering params

File: static/src/js/components/GranuleResults/GranuleResultsList.js [hunk 3]
```diff
@@ -85,6 +85,9 @@ export class GranuleResultsList extends PureComponent {
       granuleIds = difference(allGranuleIds, excludedGranuleIds)
     }
 
+    // Determine the correct granule count based on granules that have been removed
+    const granuleCount = (hits - excludedGranuleIds.length)
```
Comment (trevorlang) [inline static/src/js/components/GranuleResults/GranuleResultsList.js]: If you are now excluding the granules in the request, do you need to subtract the excluded count like this? Seems like the hits in the request should have the correct value, no?

Comment (abbottry) [reply static/src/js/components/GranuleResults/GranuleResultsList.js]: I'm not excluding them in the requests in the UI, I only add them to the params when the download is submitted and stored in the database. Those granule params are only used when displaying data for the download access methods and when the orders are submitted to CMR and ESI.

File: static/src/js/components/ProjectCollections/__tests__/ProjectHeader.test.js [hunk 4]
```diff
@@ -76,13 +56,42 @@ describe('ProjectHeader component', () => {
...
+        }
+      },
+      project: {
+        collectionIds: ['collectionId1', 'collectionId2']
+      }
+    })
+
     test('renders collection count correctly', () => {
-      const { enzymeWrapper } = setup('multi-collection')
+      // const { enzymeWrapper } = setup('multi-collection')
```
Comment (macrouch) [inline static/src/js/components/ProjectCollections/__tests__/ProjectHeader.test.js]: remove

File: static/src/js/components/ProjectCollections/__tests__/ProjectHeader.test.js [hunk 5]
```diff
@@ -76,13 +56,42 @@ describe('ProjectHeader component', () => {
...
       expect(enzymeWrapper.find('.project-header__stats-item--collections').text()).toEqual('2 Collections')
     })
 
     test('renders collection size correctly', () => {
-      const { enzymeWrapper } = setup('multi-collection')
+      // const { enzymeWrapper } = setup('multi-collection')
```
Comment (macrouch) [inline static/src/js/components/ProjectCollections/__tests__/ProjectHeader.test.js]: remove

File: static/src/js/components/ProjectCollections/__tests__/ProjectHeader.test.js [hunk 6]
```diff
@@ -94,10 +103,75 @@ describe('ProjectHeader component', () => {
...
+          }
+        }
+      },
+      project: {
+        collectionIds: ['collectionId1', 'collectionId2']
+      }
+    })
+
+    test('renders granule count correctly', () => {
+      // const { enzymeWrapper } = setup('multi-granule')
```
Comment (macrouch) [inline static/src/js/components/ProjectCollections/__tests__/ProjectHeader.test.js]: remove

**Pull request merged**